### PR TITLE
Remove usage of proprietary sun class that may be unavailable

### PR DESCRIPTION
**Pull request by isopov**

### CODE REVIEW

**Review by akudiyar — APPROVED** [review]

**Comment by isopov** [comment]
@akudiyar 
>1 workflow awaiting approval

FYI in case it is not clear - I cannot merge this PR after approve. I suppose someone is needed to approve the run of github actions and in case they will be successful I will be able to merge

**Comment by isopov** [comment]
BTW
> Only those with write access to this repository can merge pull requests.

**Comment by akudiyar** [comment]
@isopov The pipelines keep failing, we need to investigate what happens.

**Pull request merged**
